src/partials/prohibited-activities.md

```diff
@@ -0,0 +1,39 @@
+# Prohibited activities {% #prohibited-activities %}
+
+The following actions are prohibited on the Appwrite platform and may lead to immediate suspension or termination:
+
+- **Illegal Content**: Hosting, transmitting, or distributing content that is illegal, infringing, harmful, threatening, defamatory, obscene, harassing, or otherwise objectionable.
+
+- **Malware and Harmful Code**: Uploading or distributing viruses, malware, or any other malicious code intended to harm or gain unauthorized access to any system, data, or personal information.
+
+- **Unsolicited Communications**: Sending unauthorized or unsolicited advertising, spam, or any other form of improper solicitation.
+
+- **Fraudulent Activities**: Engaging in deceptive practices or activities intended to defraud, mislead, or deceive others.
+
+- **Abuse of Resources**: Misusing the Service in a way that disrupts its integrity or performance.
+
+- **Unauthorized Access**: Attempting to gain unauthorized access to the Service, other user accounts, computer systems, or networks connected to the Service.
+
+- **Crypto Mining**: Using the Service for cryptocurrency mining or similar computationally intensive activities without authorization.
+
+- **Unauthorized Load Testing**: Conducting load testing or stress testing on the Service without prior written approval.
+
+- **Penetration Testing**: Performing penetration testing or security assessments without explicit authorization.
+
+- **Denial-of-Service (DoS) Attacks**: Initiating or participating in denial-of-service attacks or other activities designed to disrupt the Service.
+
+- **Phishing Schemes**: Engaging in phishing activities or schemes designed to deceive individuals into providing sensitive information.
+
+- **Avoidance of Payments**: Attempting to circumvent payment obligations, including manipulating billing systems or using the Service without proper authorization or payment.
+
+- **Violation of Laws**: Engaging in activities that violate any applicable local, state, national, or international laws or regulations.
+
+## Prohibited Use of the SMS Messages Service
+
+You agree not to use the Service for the sending of unsolicited or spam SMS messages. This includes but is not limited to the prohibition of:
+
+- Using the Service in connection with any content, product, or service the recipient has not explicitly requested.
+
+- Sending SMS messages through the Service unless you have obtained prior express consent from the recipients, as required by applicable law.
+
+Failure to adhere to these prohibitions and any applicable laws relating to the use of the SMS Service can result in immediate termination of your account and further legal action. You shall comply with all applicable laws and regulations.
```

Comment:
Sorry, let's leave this out of the partial and we'll only include it in the Terms since it's kind of redundant.

Reply:
Oh okay 👍

Reply:
```suggestion

```

Reply:
removed